src/style-dictionary-dist/TerrawareTheme.ts

```diff
@@ -0,0 +1,831 @@
+      },
+      "opacity": {
+        "value": "0.2",
+        "filePath": "./json/system/opacity-base.json",
+        "isSource": true,
+        "original": {
+          "value": "0.2"
+        },
+        "name": "TwOpctyBase020",
+        "attributes": {
+          "category": "tw-opcty-base-020"
+        },
+        "path": [
+          "tw-opcty-base-020"
+        ]
+      }
+    },
+    "FntDlgBxMessageFontFamily": "'Inter', sans-serif",
+    "FntDlgBxMessageFontSize": "16px",
+    "FntDlgBxMessageFontWeight": 400,
+    "FntDlgBxMessageLineHeight": "24px",
+    "FntDlgBxTitleFontFamily": "'Inter', sans-serif",
+    "FntDlgBxTitleFontSize": "20px",
+    "FntDlgBxTitleFontWeight": 600,
+    "FntDlgBxTitleLineHeight": "28px",
+    "FntMssgBodyFontFamily": "'Inter', sans-serif",
+    "FntMssgBodyFontSize": "16px",
+    "FntMssgBodyFontWeight": 400,
+    "FntMssgBodyLineHeight": "24px",
+    "FntMssgTitleFontFamily": "'Inter', sans-serif",
+    "FntMssgTitleFontSize": "16px",
+    "FntMssgTitleFontWeight": 600,
+    "FntMssgTitleLineHeight": "24px",
+    "ClrNvgtnSideNavItemFill": "#F9FAFA",
+    "ClrNvgtnSideNavItemFillActive": "#EDF0F1",
+    "ClrNvgtnSideNavItemFillHover": "#F2F4F5",
+    "ClrNvgtnSideNavItemIndicatorFill": "#0067C8",
+    "ClrNvgtnSideNavItemLabel": "#3A4445",
+    "ClrNvgtnSideNavSectionFill": "#F9FAFA",
+    "ClrNvgtnSideNavSectionTitle": "#3A4445",
+    "FntNvgtnSideNavItemLabelFontFamily": "'Inter', sans-serif",
+    "FntNvgtnSideNavItemLabelFontSize": "14px",
+    "FntNvgtnSideNavItemLabelFontWeight": 500,
+    "FntNvgtnSideNavItemLabelLineHeight": "16px",
+    "FntNvgtnSideNavSectionTitleFontFamily": "'Inter', sans-serif",
+    "FntNvgtnSideNavSectionTitleFontSize": "12px",
+    "FntNvgtnSideNavSectionTitleFontWeight": 400,
+    "FntNvgtnSideNavSectionTitleLineHeight": "16px",
+    "FntNvgtnSideNavSectionTitleTextTransform": "uppercase",
+    "SzNvgtnSideNavItemIndicator": "4px",
+    "SzNvgtnSideNavWidth": "200px",
+    "FntPrgrssCircleValueLargeFontFamily": "'Inter', sans-serif",
+    "FntPrgrssCircleValueLargeFontSize": "48px",
+    "FntPrgrssCircleValueLargeFontWeight": 400,
+    "FntPrgrssCircleValueLargeLineHeight": "60px",
+    "FntPrgrssCircleValueMediumFontFamily": "'Inter', sans-serif",
+    "FntPrgrssCircleValueMediumFontSize": "24px",
+    "FntPrgrssCircleValueMediumFontWeight": 400,
+    "FntPrgrssCircleValueMediumLineHeight": "32px",
+    "FntPrgrssCircleValueSmallFontFamily": "'Inter', sans-serif",
+    "FntPrgrssCircleValueSmallFontSize": "12px",
+    "FntPrgrssCircleValueSmallFontWeight": 400,
+    "FntPrgrssCircleValueSmallLineHeight": "16px",
+    "ClrDvdrFill": "#A9B7B8",
+    "ClrIconFillCritical": "#FE0003",
+    "ClrIconFillPassive": "#708284",
+    "ClrIconFillProductive": "#007DF2",
+    "ClrIconFillSuccess": "#308F5F",
+    "ClrIconFillText": "#3A4445",
+    "ClrIconFillTextInv": "#FFFFFF",
+    "ClrIconFillWarning": "#BD6931",
+    "ClrBaseBlack": "#000000",
+    "ClrBaseGray025": "#F9FAFA",
```

Comment:
Are we going to have all the color in pallete and in terraware also?

Reply:
for now yes, i will clean it up once we get the new tokens - since they won't be starting with Clr.